src/components/admin/event/ManageEventCard/style.module.scss

```diff
@@ -0,0 +1,27 @@
+.event {
+  align-items: center;
+  border-radius: 8px;
+  box-shadow: 0 0 4px var(--theme-primary-6);
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  min-height: 6rem;
+  padding: 1rem;
+
+  .coverContainer {
+    height: 3rem;
+    position: relative;
+    width: 100%;
+
+    .cover {
+      height: auto;
+      object-fit: cover;
+      width: 100%;
+    }
+  }
+
+  a,
+  button {
+    width: fit-content;
+  }
```

Comment:
What is this rule for? It doesn't look necessary to me

Reply:
Hmm removing it doesn't seem to do anything so I took it out.